bundles/org.openhab.binding.rotel/src/main/java/org/openhab/binding/rotel/internal/handler/RotelHandler.java

```diff
@@ -798,6 +831,10 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                         if (!isPowerOn()) {
                             success = false;
                             logger.debug("Command {} from channel {} ignored: device in standby", command, channel);
+                        } else if (OnOffType.from(tcbypass) == OnOffType.ON) {
+                            logger.debug("Command {} from channel {} ignored: device in state TCBYPASS ON", command,
```

Comment:
I rather suggest: "Command {} from channel {} ignored: ton control bypass is ON"

Reply:
fixed